.vscode/launch.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
Curious if you can give me a quick demo at some point, I've never used the advanced config stuff in vscode.

Reply:
I don't know a lot about it, I'm learning it as I need to, but I can share for sure 👍